Sources/Private/LayerContainers/AnimationContainer.swift

```diff
@@ -134,7 +134,9 @@ final class AnimationContainer: CALayer, RootAnimationLayer {
     if respectAnimationFrameRate {
       newFrame = floor(newFrame)
     }
-    animationLayers.forEach( { $0.displayWithFrame(frame: newFrame, forceUpdates: false) })
+    animationLayers.forEach {
+      $0.displayWithFrame(frame: newFrame, forceUpdates: false)
+    }
```

Comment:
nit: `forEach` feels non-idiomatic here, especially when used with `$0`
```suggestion
    for layer in animationLayers {
      layer.displayWithFrame(frame: newFrame, forceUpdates: false)
    }
```